diff -Nur calamares-3.2.11-default-settings/calamares.desktop calamares-3.2.11-kdesu/calamares.desktop
--- calamares-3.2.11-default-settings/calamares.desktop	2019-07-06 00:52:35.000000000 +0200
+++ calamares-3.2.11-kdesu/calamares.desktop	2019-07-08 23:54:03.108978532 +0200
@@ -5,7 +5,7 @@
 GenericName=System Installer
 Keywords=calamares;system;installer
 TryExec=calamares
-Exec=pkexec /usr/bin/calamares
+Exec=kdesu /usr/bin/calamares
 Comment=Calamares — System Installer
 Icon=calamares
 Terminal=false
diff -Nur calamares-3.2.11-default-settings/calamares.desktop.in calamares-3.2.11-kdesu/calamares.desktop.in
--- calamares-3.2.11-default-settings/calamares.desktop.in	2019-07-06 00:52:35.000000000 +0200
+++ calamares-3.2.11-kdesu/calamares.desktop.in	2019-07-08 23:54:03.269977444 +0200
@@ -5,7 +5,7 @@
 GenericName=System Installer
 Keywords=calamares;system;installer
 TryExec=calamares
-Exec=pkexec /usr/bin/calamares
+Exec=kdesu /usr/bin/calamares
 Comment=Calamares — System Installer
 Icon=calamares
 Terminal=false
diff -Nur calamares-3.2.11-default-settings/CMakeLists.txt calamares-3.2.11-kdesu/CMakeLists.txt
--- calamares-3.2.11-default-settings/CMakeLists.txt	2019-07-08 22:25:32.488871708 +0200
+++ calamares-3.2.11-kdesu/CMakeLists.txt	2019-07-08 23:54:03.269977444 +0200
@@ -45,7 +45,6 @@
 ### OPTIONS
 #
 option( INSTALL_CONFIG "Install configuration files" ON )
-option( INSTALL_POLKIT "Install Polkit configuration" ON )
 option( BUILD_TESTING "Build the testing tree." ON )
 option( WITH_PYTHON "Enable Python modules API (requires Boost.Python)." ON )
 option( WITH_PYTHONQT "Enable next generation Python modules API (experimental, requires PythonQt)." OFF )
@@ -248,18 +247,6 @@
 #
 find_package( Qt5 ${QT_VERSION} CONFIG REQUIRED Concurrent Core Gui Widgets LinguistTools Svg Quick QuickWidgets )
 find_package( YAMLCPP ${YAMLCPP_VERSION} REQUIRED )
-if( INSTALL_POLKIT )
-    find_package( PolkitQt5-1 REQUIRED )
-else()
-    # Find it anyway, for dependencies-reporting
-    find_package( PolkitQt5-1 )
-endif()
-set_package_properties(
-    PolkitQt5-1 PROPERTIES
-    DESCRIPTION "Qt5 support for Polkit"
-    URL "https://cgit.kde.org/polkit-qt-1.git"
-    PURPOSE "PolkitQt5-1 helps with installing Polkit configuration"
-)
 
 # Find ECM once, and add it to the module search path; Calamares
 # modules that need ECM can do
@@ -539,15 +526,6 @@
     )
 endif()
 
-if( INSTALL_POLKIT )
-    install(
-        FILES
-            com.github.calamares.calamares.policy
-        DESTINATION
-          "${POLKITQT-1_POLICY_FILES_INSTALL_DIR}"
-    )
-endif()
-
 install(
   FILES
       calamares.desktop
diff -Nur calamares-3.2.11-default-settings/com.github.calamares.calamares.policy calamares-3.2.11-kdesu/com.github.calamares.calamares.policy
--- calamares-3.2.11-default-settings/com.github.calamares.calamares.policy	2019-07-06 00:52:35.000000000 +0200
+++ calamares-3.2.11-kdesu/com.github.calamares.calamares.policy	1970-01-01 01:00:00.000000000 +0100
@@ -1,22 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE policyconfig PUBLIC
-"-//freedesktop//DTD PolicyKit Policy Configuration 1.0//EN"
-"http://www.freedesktop.org/standards/PolicyKit/1/policyconfig.dtd">
-<policyconfig>
-
- <vendor>Calamares</vendor>
- <vendor_url>https://github.com/calamares</vendor_url>
-
- <action id="com.github.calamares.calamares.pkexec.run">
-    <description>Run Installer</description>
-    <message>Authentication is required to run the installation program</message>
-    <icon_name>drive-harddisk</icon_name>
-    <defaults>
-     <allow_any>no</allow_any>
-     <allow_inactive>no</allow_inactive>
-     <allow_active>auth_admin</allow_active>
-    </defaults>
-    <annotate key="org.freedesktop.policykit.exec.path">/usr/bin/calamares</annotate>
-    <annotate key="org.freedesktop.policykit.exec.allow_gui">true</annotate>
- </action>
-</policyconfig>
\ No newline at end of file
