diff -ur calamares-8a623cc1181e94b9bf4f1511061ef817e4ccc5df/src/modules/grub/main.py calamares-grub2-tools/src/modules/grub/main.py
--- calamares-8a623cc1181e94b9bf4f1511061ef817e4ccc5df/src/modules/grub/main.py	2014-10-17 11:52:29.000000000 +0200
+++ calamares-grub2-tools/src/modules/grub/main.py	2014-10-20 23:49:46.000000000 +0200
@@ -23,8 +23,8 @@
 
 def install_grub(boot_loader):
     install_path = boot_loader["installPath"]
-    check_chroot_call(["grub-install", install_path])
-    check_chroot_call(["grub-mkconfig", "-o", "/boot/grub/grub.cfg"])
+    check_chroot_call(["grub2-install", install_path])
+    check_chroot_call(["grub2-mkconfig", "-o", "/boot/grub2/grub.cfg"])
 
 
 def run():
