diff -ur calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/settings.conf calamares-default-settings/settings.conf
--- calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/settings.conf	2014-12-06 10:07:54.000000000 +0100
+++ calamares-default-settings/settings.conf	2014-12-07 03:18:42.000000000 +0100
@@ -4,7 +4,7 @@
 # Modules can be job modules (with different interfaces) and QtWidgets view modules.
 # They could all be placed in a number of different paths.
 # "local" is LIBDIR/calamares/modules with settings in SHARE/calamares/modules
-modules-search: [ local, /path/to/dir/with/more/modules ]
+modules-search: [ local ]
 
 # We define the module names in the order they should show up (QtWidget view modules,
 # with one or more pages) OR be executed if enqueued (all other modules).
@@ -39,14 +39,14 @@
 - locale
 - keyboard
 - localecfg
-- initcpiocfg
-- initcpio
+#- initcpiocfg
+#- initcpio
 - users
-#- displaymanager
+- displaymanager
 - networkcfg
 - hwclock
 - services
-#- dracut
+- dracut
 #- initramfs
 - grubcfg
 - grub
@@ -67,4 +67,4 @@
 # paths relative to the current directory.
 # Only the name of the branding component (directory) should be specified here, Calamares
 # then takes care of finding it and loading the contents.
-branding: default
+branding: auto
diff -ur calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/src/modules/grub/grub.conf calamares-default-settings/src/modules/grub/grub.conf
--- calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/src/modules/grub/grub.conf	2014-12-06 10:07:54.000000000 +0100
+++ calamares-default-settings/src/modules/grub/grub.conf	2014-12-07 03:18:42.000000000 +0100
@@ -1,11 +1,13 @@
 ---
 # GRUB 2 binary names and boot directory
 # Some distributions (e.g. Fedora) use grub2-* (resp. /boot/grub2/) names.
-grubInstall: "grub-install"
-grubMkconfig: "grub-mkconfig"
-grubCfg: "/boot/grub/grub.cfg"
+grubInstall: "grub2-install"
+grubMkconfig: "grub2-mkconfig"
+grubCfg: "/boot/grub2/grub.cfg"
 # Optionally set the --bootloader-id to use for EFI. If not set, this defaults
 # to the bootloaderEntryName from branding.desc with problematic characters
 # replaced. If an efiBootloaderId is specified here, it is taken to already be a
 # valid directory name, so no such postprocessing is done in this case.
-# efiBootloaderId: "dirname"
+# PACKAGER NOTE: If you really want to change this value, you will also have to
+#                change the hardcoded directory name in the grub2(-efi) package.
+efiBootloaderId: "fedora"
diff -ur calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/src/modules/locale/locale.conf calamares-default-settings/src/modules/locale/locale.conf
--- calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/src/modules/locale/locale.conf	2014-12-06 10:07:54.000000000 +0100
+++ calamares-default-settings/src/modules/locale/locale.conf	2014-12-07 03:18:42.000000000 +0100
@@ -1,3 +1,4 @@
 ---
-region:     "Europe"
-zone:       "London"
+# default to UTC
+region:     "Atlantic"
+zone:       "Reykjavik"
diff -ur calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/src/modules/unpackfs/unpackfs.conf calamares-default-settings/src/modules/unpackfs/unpackfs.conf
--- calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/src/modules/unpackfs/unpackfs.conf	2014-12-06 10:07:54.000000000 +0100
+++ calamares-default-settings/src/modules/unpackfs/unpackfs.conf	2014-12-07 03:18:42.000000000 +0100
@@ -1,8 +1,5 @@
 ---
 unpack:
-    -   source: "/path/to/filesystem.img"
+    -   source: "/dev/mapper/live-base"
         sourcefs: "ext4"
         destination: ""
-    -   source: "/path/to/another/filesystem.sqfs"
-        sourcefs: "squashfs"
-        destination: ""
diff -ur calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/src/modules/users/users.conf calamares-default-settings/src/modules/users/users.conf
--- calamares-eb748cca8ebfc77c04c12c35a7aefac2b55af26f/src/modules/users/users.conf	2014-12-06 10:07:54.000000000 +0100
+++ calamares-default-settings/src/modules/users/users.conf	2014-12-07 03:18:42.000000000 +0100
@@ -3,8 +3,7 @@
 defaultGroups:
     - lp
     - video
-    - network
-    - storage
+    - dialout
     - wheel
     - audio
 autologinGroup: autologin
