diff -ur calamares-1.0.1/src/modules/finished/FinishedPage.cpp calamares-1.0.1-fix-reboot/src/modules/finished/FinishedPage.cpp
--- calamares-1.0.1/src/modules/finished/FinishedPage.cpp	2015-01-31 16:06:54.000000000 +0100
+++ calamares-1.0.1-fix-reboot/src/modules/finished/FinishedPage.cpp	2015-02-05 02:00:37.000000000 +0100
@@ -90,7 +90,7 @@
         {
             if ( ui->restartCheckBox->isVisible() &&
                  ui->restartCheckBox->isChecked() )
-                QProcess::execute( "/bin/sh", { m_restartNowCommand } );
+                QProcess::execute( "/bin/sh", { "-c", m_restartNowCommand } );
         } );
     }
 }
