diff -ur calamares-3.1.8/settings.conf calamares-3.1.8-default-settings/settings.conf
--- calamares-3.1.8/settings.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/settings.conf	2017-11-14 16:53:50.562464805 +0100
@@ -80,21 +80,22 @@
   - locale
   - keyboard
   - localecfg
-#  - luksbootkeyfile
+  - luksbootkeyfile
 #  - luksopenswaphookcfg
-#  - dracutlukscfg
-#  - plymouthcfg
-  - initcpiocfg
-  - initcpio
+  - dracutlukscfg
+  - plymouthcfg
+#  - initcpiocfg
+#  - initcpio
   - users
   - displaymanager
   - networkcfg
   - hwclock
   - services
-#  - dracut
-  - initramfs
-#  - grubcfg
+  - dracut
+#  - initramfs
+  - grubcfg
   - bootloader
+  - packages
   - umount
 - show:
 #  - webview@owncloud
@@ -109,7 +110,7 @@
 # Only the name of the branding component (directory) should be specified here, Calamares
 # then takes care of finding it and loading the contents.
 # YAML: string.
-branding: default
+branding: auto
 
 # If this is set to true, Calamares will show an "Are you sure?" prompt right before
 # each execution phase, i.e. at points of no return. If this is set to false, no prompt
diff -ur calamares-3.1.8/src/modules/bootloader/bootloader.conf calamares-3.1.8-default-settings/src/modules/bootloader/bootloader.conf
--- calamares-3.1.8/src/modules/bootloader/bootloader.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/bootloader/bootloader.conf	2017-11-14 16:53:50.641463674 +0100
@@ -17,11 +17,13 @@
 
 # GRUB 2 binary names and boot directory
 # Some distributions (e.g. Fedora) use grub2-* (resp. /boot/grub2/) names.
-grubInstall: "grub-install"
-grubMkconfig: "grub-mkconfig"
-grubCfg: "/boot/grub/grub.cfg"
+grubInstall: "grub2-install"
+grubMkconfig: "grub2-mkconfig"
+grubCfg: "/boot/grub2/grub.cfg"
 # Optionally set the --bootloader-id to use for EFI. If not set, this defaults
 # to the bootloaderEntryName from branding.desc with problematic characters
 # replaced. If an efiBootloaderId is specified here, it is taken to already be a
 # valid directory name, so no such postprocessing is done in this case.
-# efiBootloaderId: "dirname"
+# PACKAGER NOTE: If you really want to change this value, you will also have to
+#                change the hardcoded directory name in the grub2(-efi) package.
+efiBootloaderId: "fedora"
diff -ur calamares-3.1.8/src/modules/keyboard/keyboard.conf calamares-3.1.8-default-settings/src/modules/keyboard/keyboard.conf
--- calamares-3.1.8/src/modules/keyboard/keyboard.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/keyboard/keyboard.conf	2017-11-14 16:53:50.721462529 +0100
@@ -11,4 +11,4 @@
 # Write keymap configuration to /etc/default/keyboard, usually
 # found on Debian-related systems.
 # Defaults to true if nothing is set.
-#writeEtcDefaultKeyboard:   true
+writeEtcDefaultKeyboard: false
diff -ur calamares-3.1.8/src/modules/license/license.conf calamares-3.1.8-default-settings/src/modules/license/license.conf
--- calamares-3.1.8/src/modules/license/license.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/license/license.conf	2017-11-14 16:53:50.721462529 +0100
@@ -3,21 +3,9 @@
 ---
 # YAML: list of maps.
 entries:
-- id:       nvidia      # Entry identifier, must be unique. Not user visible. YAML: string.
-  name:     Nvidia      # Pretty name for the software product, user visible and untranslatable. YAML: string.
-  vendor:   Nvidia Corporation # Pretty name for the software vendor, user visible and untranslatable. YAML: string, optional, default is empty.
+- id:       evil        # Entry identifier, must be unique. Not user visible. YAML: string.
+  name:     Evil        # Pretty name for the software product, user visible and untranslatable. YAML: string.
+  vendor:   Evil Corporation # Pretty name for the software vendor, user visible and untranslatable. YAML: string, optional, default is empty.
   type:     driver      # Package type for presentation, not user visible but affects user visible strings. YAML: string, allowed values: driver, gpudriver, browserplugin, codec, package, software; optional, default is software.
-  url:      http://developer.download.nvidia.com/cg/Cg_3.0/license.pdf  # Url of license text to display in a web view. YAML: string.
+  url:      http://www.example.com/evil_666.0/license.pdf  # Url of license text to display in a web view. YAML: string.
   required: false       # If set to true, the user cannot proceed without accepting this license. YAML: boolean, optional, default is false.
-- id:       amd
-  name:     Catalyst
-  vendor:   "Advanced Micro Devices, Inc."
-  type:     gpudriver
-  url:      http://support.amd.com/en-us/download/eula
-  required: false
-- id:       flashplugin
-  name:     Adobe Flash
-  vendor:   Adobe Systems Incorporated
-  type:     browserplugin
-  url:      http://www.adobe.com/products/eulas/pdfs/PlatformClients_PC_WWEULA_Combined_20100108_1657.pdf
-  required: true
diff -ur calamares-3.1.8/src/modules/netinstall/netinstall.conf calamares-3.1.8-default-settings/src/modules/netinstall/netinstall.conf
--- calamares-3.1.8/src/modules/netinstall/netinstall.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/netinstall/netinstall.conf	2017-11-14 16:53:50.787461586 +0100
@@ -1,7 +1,10 @@
 ---
 # This is the URL that is retrieved to get the netinstall groups-and-packages
 # data (which should be in the format described in netinstall.yaml).
-groupsUrl: http://chakraos.org/netinstall.php
+# Packager note: If you want to enable the netinstall module, you will also have
+#                to create such a file (see the example in the source code).
+groupsUrl: file:///etc/calamares/modules/netinstall.yaml
+#groupsUrl: http://example.com/netinstall.yaml
 
 # If the installation can proceed without netinstall (e.g. the Live CD
 # can create a working installed system, but netinstall is preferred
diff -ur calamares-3.1.8/src/modules/packages/packages.conf calamares-3.1.8-default-settings/src/modules/packages/packages.conf
--- calamares-3.1.8/src/modules/packages/packages.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/packages/packages.conf	2017-11-14 16:53:50.788461571 +0100
@@ -12,12 +12,12 @@
 #  - entropy     - Sabayon package manager
 #  - dummy       - Dummy manager, only logs
 #
-backend: dummy
+backend: dnf
 
 # If set to true, a package-manager specific update procedure
 # is run first (only if there is internet) to update the list
 # of packages and dependencies.
-update_db: true
+update_db: false
 
 #
 # List of maps with package operations such as install or remove.
@@ -112,11 +112,5 @@
 # of packages, this can lead to a considerable time savings.
 #
 operations:
-  - install:
-    - vi
-    - wget
-    - binutils
-  - remove:
-    - vi
-    - wget
-    - binutils
+  - try_remove:
+    - calamares
diff -ur calamares-3.1.8/src/modules/partition/partition.conf calamares-3.1.8-default-settings/src/modules/partition/partition.conf
--- calamares-3.1.8/src/modules/partition/partition.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/partition/partition.conf	2017-11-14 16:53:50.862460511 +0100
@@ -21,7 +21,7 @@
 neverCreateSwap:        false
 
 # Correctly draw nested (e.g. logical) partitions as such.
-drawNestedPartitions:   false
+drawNestedPartitions:   true
 
 # Show/hide partition labels on manual partitioning page.
 alwaysShowPartitionLabels: true
diff -ur calamares-3.1.8/src/modules/services/services.conf calamares-3.1.8-default-settings/src/modules/services/services.conf
--- calamares-3.1.8/src/modules/services/services.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/services/services.conf	2017-11-14 16:53:50.863460497 +0100
@@ -12,9 +12,4 @@
   - name: "graphical"
     mandatory: true
 
-disable:
-  - name: "pacman-init"
-    mandatory: false
-
-# Example to express an empty list:
-# disable: []
+disable: []
diff -ur calamares-3.1.8/src/modules/umount/umount.conf calamares-3.1.8-default-settings/src/modules/umount/umount.conf
--- calamares-3.1.8/src/modules/umount/umount.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/umount/umount.conf	2017-11-14 16:53:50.863460497 +0100
@@ -1,9 +1,6 @@
 ---
-#srcLog:       "/path/to/installation.log"
-#destLog:      "/var/log/installation.log"
-# example when using the Calamares created log:
-#srcLog:      "/root/.cache/Calamares/Calamares/Calamares.log"
-#destLog:     "/var/log/Calamares.log"
+srcLog:      "/root/.cache/Calamares/Calamares/Calamares.log"
+destLog:     "/var/log/calamares.log"
 # example when creating with a sudo calamares -d log:
-#srcLog:      "/home/live/installation.log"
+#srcLog:      "/home/liveuser/installation.log"
 #destLog:     "/var/log/installation.log"
diff -ur calamares-3.1.8/src/modules/unpackfs/unpackfs.conf calamares-3.1.8-default-settings/src/modules/unpackfs/unpackfs.conf
--- calamares-3.1.8/src/modules/unpackfs/unpackfs.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/unpackfs/unpackfs.conf	2017-11-14 16:53:50.863460497 +0100
@@ -33,6 +33,10 @@
 
 # You can list filesystem source paths relative to the Calamares run
 # directory, if you use -d (this is only useful for testing, though).
-    -   source: ./example.sqfs
-        sourcefs: squashfs
+#    -   source: ./example.sqfs
+#        sourcefs: squashfs
+#        destination: ""
+
+    -   source: "/dev/mapper/live-base"
+        sourcefs: "ext4"
         destination: ""
diff -ur calamares-3.1.8/src/modules/users/users.conf calamares-3.1.8-default-settings/src/modules/users/users.conf
--- calamares-3.1.8/src/modules/users/users.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/users/users.conf	2017-11-14 16:53:50.863460497 +0100
@@ -15,15 +15,15 @@
     - users
     - lp
     - video
-    - network
-    - storage
+    - dialout
     - wheel
     - audio
 autologinGroup:  autologin
-doAutologin:     true
+doAutologin:     false
 
-# remove the following line to avoid creating /etc/sudoers.d/10-installer
-sudoersGroup:    wheel
+# this is already the default in /etc/sudoers, so do NOT set it here, as that
+# would create a confusing /etc/sudoers.d/10-installer file that sets it again
+# sudoersGroup:    wheel
 
 setRootPassword: true
 doReusePassword: true
diff -ur calamares-3.1.8/src/modules/welcome/welcome.conf calamares-3.1.8-default-settings/src/modules/welcome/welcome.conf
--- calamares-3.1.8/src/modules/welcome/welcome.conf	2017-11-14 16:18:58.000000000 +0100
+++ calamares-3.1.8-default-settings/src/modules/welcome/welcome.conf	2017-11-14 16:53:50.864460483 +0100
@@ -6,7 +6,7 @@
 requirements:
     requiredStorage:    5.5
     requiredRam:        1.0
-    internetCheckUrl:   http://google.com
+    internetCheckUrl:   http://fedoraproject.org/static/hotspot.txt
 
     # List conditions to check. Each listed condition will be
     # probed in some way, and yields true or false according to
