diff -ur calamares-1.1.3/settings.conf calamares-1.1.3-default-settings/settings.conf
--- calamares-1.1.3/settings.conf	2015-09-14 17:48:01.000000000 +0200
+++ calamares-1.1.3-default-settings/settings.conf	2015-09-26 01:24:33.164575482 +0200
@@ -20,12 +20,12 @@
 # Jobs should be executed sparingly (if at all) in this phase.
 prepare:
 - welcome
-- license
+#- license
 - locale
 - keyboard
 - partition
 - users
-- webview
+#- webview
 - summary
 
 # Phase 2 - install.
@@ -43,17 +43,18 @@
 - locale
 - keyboard
 - localecfg
-- initcpiocfg
-- initcpio
+#- initcpiocfg
+#- initcpio
 - users
 - displaymanager
 - networkcfg
 - hwclock
 - services
-#- dracut
+- dracut
 #- initramfs
 - grubcfg
 - bootloader
+- packages
 - umount
 
 # Phase 3 - postinstall.
@@ -70,7 +71,7 @@
 # as paths relative to the current directory.
 # Only the name of the branding component (directory) should be specified here, Calamares
 # then takes care of finding it and loading the contents.
-branding: default
+branding: auto
 
 # If this is set to true, Calamares will show an "are you sure?" prompt right before the
 # install phase, i.e. at the point of no return. If it's false, no prompt is shown.
diff -ur calamares-1.1.3/src/modules/bootloader/bootloader.conf calamares-1.1.3-default-settings/src/modules/bootloader/bootloader.conf
--- calamares-1.1.3/src/modules/bootloader/bootloader.conf	2015-08-06 11:58:32.000000000 +0200
+++ calamares-1.1.3-default-settings/src/modules/bootloader/bootloader.conf	2015-09-26 01:23:55.545357323 +0200
@@ -17,11 +17,13 @@
 
 # GRUB 2 binary names and boot directory
 # Some distributions (e.g. Fedora) use grub2-* (resp. /boot/grub2/) names.
-grubInstall: "grub-install"
-grubMkconfig: "grub-mkconfig"
-grubCfg: "/boot/grub/grub.cfg"
+grubInstall: "grub2-install"
+grubMkconfig: "grub2-mkconfig"
+grubCfg: "/boot/grub2/grub.cfg"
 # Optionally set the --bootloader-id to use for EFI. If not set, this defaults
 # to the bootloaderEntryName from branding.desc with problematic characters
 # replaced. If an efiBootloaderId is specified here, it is taken to already be a
 # valid directory name, so no such postprocessing is done in this case.
-# efiBootloaderId: "dirname"
+# PACKAGER NOTE: If you really want to change this value, you will also have to
+#                change the hardcoded directory name in the grub2(-efi) package.
+efiBootloaderId: "fedora"
diff -ur calamares-1.1.3/src/modules/license/license.conf calamares-1.1.3-default-settings/src/modules/license/license.conf
--- calamares-1.1.3/src/modules/license/license.conf	2015-09-14 16:51:04.000000000 +0200
+++ calamares-1.1.3-default-settings/src/modules/license/license.conf	2015-09-26 01:44:24.149437908 +0200
@@ -3,21 +3,9 @@
 ---
 # YAML: list of maps.
 entries:
-- id:       nvidia      # Entry identifier, must be unique. Not user visible. YAML: string.
-  name:     Nvidia      # Pretty name for the software product, user visible and untranslatable. YAML: string.
-  vendor:   Nvidia Corporation # Pretty name for the software vendor, user visible and untranslatable. YAML: string, optional, default is empty.
+- id:       evil        # Entry identifier, must be unique. Not user visible. YAML: string.
+  name:     Evil        # Pretty name for the software product, user visible and untranslatable. YAML: string.
+  vendor:   Evil Corporation # Pretty name for the software vendor, user visible and untranslatable. YAML: string, optional, default is empty.
   type:     driver      # Package type for presentation, not user visible but affects user visible strings. YAML: string, allowed values: driver, gpudriver, browserplugin, codec, package, software; optional, default is software.
-  url:      http://developer.download.nvidia.com/cg/Cg_3.0/license.pdf  # Url of license text to display in a web view. YAML: string.
+  url:      http://www.example.com/evil_666.0/license.pdf  # Url of license text to display in a web view. YAML: string.
   required: false       # If set to true, the user cannot proceed without accepting this license. YAML: boolean, optional, default is false.
-- id:       amd
-  name:     Catalyst
-  vendor:   "Advanced Micro Devices, Inc."
-  type:     gpudriver
-  url:      http://support.amd.com/en-us/download/eula
-  required: false
-- id:       flashplugin
-  name:     Adobe Flash
-  vendor:   Adobe Systems Incorporated
-  type:     browserplugin
-  url:      http://www.adobe.com/products/eulas/pdfs/PlatformClients_PC_WWEULA_Combined_20100108_1657.pdf
-  required: true
diff -ur calamares-1.1.3/src/modules/locale/locale.conf calamares-1.1.3-default-settings/src/modules/locale/locale.conf
--- calamares-1.1.3/src/modules/locale/locale.conf	2015-08-06 11:58:34.000000000 +0200
+++ calamares-1.1.3-default-settings/src/modules/locale/locale.conf	2015-09-26 01:23:55.580357527 +0200
@@ -1,3 +1,4 @@
 ---
-region:     "Europe"
-zone:       "London"
+# default to UTC
+region:     "Atlantic"
+zone:       "Reykjavik"
diff -ur calamares-1.1.3/src/modules/packages/packages.conf calamares-1.1.3-default-settings/src/modules/packages/packages.conf
--- calamares-1.1.3/src/modules/packages/packages.conf	2015-08-06 11:58:32.000000000 +0200
+++ calamares-1.1.3-default-settings/src/modules/packages/packages.conf	2015-09-26 01:23:55.609357696 +0200
@@ -11,7 +11,7 @@
 #  - portage	 - Gentoo package manager
 #  - entropy	 - Sabayon package manager
 #
-backend: packagekit
+backend: dnf
 #
 # List of maps with package operations such as install or remove.
 # Distro developers can provide a list of packages to remove
@@ -26,20 +26,7 @@
 # storage called "packageOperations" and it is processed
 # after the static list in the job configuration.
 #
-#operations:
+operations:
+  - remove:
+    - calamares
 #  - install:
-#      - pkg1
-#      - pkg2
-#  - remove:
-#      - pkg3
-#      - pkg4
-#  - install:
-#      - pkg5
-#  - remove:
-#      - pkg2
-#      - pkg1
-#    install:
-#      - pkgs6
-#      - pkg7
-#  - localInstall:
-#      - /path/to/pkg8
diff -ur calamares-1.1.3/src/modules/services/services.conf calamares-1.1.3-default-settings/src/modules/services/services.conf
--- calamares-1.1.3/src/modules/services/services.conf	2015-09-10 13:18:54.000000000 +0200
+++ calamares-1.1.3-default-settings/src/modules/services/services.conf	2015-09-26 01:42:47.715883986 +0200
@@ -12,6 +12,4 @@
   - name: "graphical"
     mandatory: true
 
-disable:
-  - name: "pacman-init"
-    mandatory: false
+disable: []
diff -ur calamares-1.1.3/src/modules/unpackfs/unpackfs.conf calamares-1.1.3-default-settings/src/modules/unpackfs/unpackfs.conf
--- calamares-1.1.3/src/modules/unpackfs/unpackfs.conf	2015-08-06 11:58:32.000000000 +0200
+++ calamares-1.1.3-default-settings/src/modules/unpackfs/unpackfs.conf	2015-09-26 01:23:55.630357818 +0200
@@ -1,8 +1,5 @@
 ---
 unpack:
-    -   source: "/path/to/filesystem.img"
+    -   source: "/dev/mapper/live-base"
         sourcefs: "ext4"
         destination: ""
-    -   source: "/path/to/another/filesystem.sqfs"
-        sourcefs: "squashfs"
-        destination: ""
diff -ur calamares-1.1.3/src/modules/users/users.conf calamares-1.1.3-default-settings/src/modules/users/users.conf
--- calamares-1.1.3/src/modules/users/users.conf	2015-08-19 12:52:13.000000000 +0200
+++ calamares-1.1.3-default-settings/src/modules/users/users.conf	2015-09-26 01:25:56.574056337 +0200
@@ -3,11 +3,10 @@
 defaultGroups:
     - lp
     - video
-    - network
-    - storage
+    - dialout
     - wheel
     - audio
 autologinGroup:  autologin
-doAutologin:     true
+doAutologin:     false
 sudoersGroup:    wheel
 setRootPassword: true
