diff -ur calamares-2.4.1/settings.conf calamares-2.4.1-default-settings/settings.conf
--- calamares-2.4.1/settings.conf	2016-08-17 15:47:21.000000000 +0200
+++ calamares-2.4.1-default-settings/settings.conf	2016-09-19 00:39:06.609469896 +0200
@@ -79,17 +79,18 @@
   - localecfg
 #  - luksbootkeyfile
 #  - luksopenswaphookcfg
-  - initcpiocfg
-  - initcpio
+#  - initcpiocfg
+#  - initcpio
   - users
   - displaymanager
   - networkcfg
   - hwclock
   - services
-#  - dracut
-  - initramfs
-#  - grubcfg
+  - dracut
+#  - initramfs
+  - grubcfg
   - bootloader
+  - packages
   - umount
 - show:
 #  - webview@owncloud
@@ -104,7 +105,7 @@
 # Only the name of the branding component (directory) should be specified here, Calamares
 # then takes care of finding it and loading the contents.
 # YAML: string.
-branding: default
+branding: auto
 
 # If this is set to true, Calamares will show an "Are you sure?" prompt right before
 # each execution phase, i.e. at points of no return. If this is set to false, no prompt
diff -ur calamares-2.4.1/src/modules/bootloader/bootloader.conf calamares-2.4.1-default-settings/src/modules/bootloader/bootloader.conf
--- calamares-2.4.1/src/modules/bootloader/bootloader.conf	2016-04-12 11:23:47.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/bootloader/bootloader.conf	2016-09-19 00:35:05.067177280 +0200
@@ -17,11 +17,13 @@
 
 # GRUB 2 binary names and boot directory
 # Some distributions (e.g. Fedora) use grub2-* (resp. /boot/grub2/) names.
-grubInstall: "grub-install"
-grubMkconfig: "grub-mkconfig"
-grubCfg: "/boot/grub/grub.cfg"
+grubInstall: "grub2-install"
+grubMkconfig: "grub2-mkconfig"
+grubCfg: "/boot/grub2/grub.cfg"
 # Optionally set the --bootloader-id to use for EFI. If not set, this defaults
 # to the bootloaderEntryName from branding.desc with problematic characters
 # replaced. If an efiBootloaderId is specified here, it is taken to already be a
 # valid directory name, so no such postprocessing is done in this case.
-# efiBootloaderId: "dirname"
+# PACKAGER NOTE: If you really want to change this value, you will also have to
+#                change the hardcoded directory name in the grub2(-efi) package.
+efiBootloaderId: "fedora"
diff -ur calamares-2.4.1/src/modules/license/license.conf calamares-2.4.1-default-settings/src/modules/license/license.conf
--- calamares-2.4.1/src/modules/license/license.conf	2016-04-12 11:23:47.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/license/license.conf	2016-09-19 00:35:05.067177280 +0200
@@ -3,21 +3,9 @@
 ---
 # YAML: list of maps.
 entries:
-- id:       nvidia      # Entry identifier, must be unique. Not user visible. YAML: string.
-  name:     Nvidia      # Pretty name for the software product, user visible and untranslatable. YAML: string.
-  vendor:   Nvidia Corporation # Pretty name for the software vendor, user visible and untranslatable. YAML: string, optional, default is empty.
+- id:       evil        # Entry identifier, must be unique. Not user visible. YAML: string.
+  name:     Evil        # Pretty name for the software product, user visible and untranslatable. YAML: string.
+  vendor:   Evil Corporation # Pretty name for the software vendor, user visible and untranslatable. YAML: string, optional, default is empty.
   type:     driver      # Package type for presentation, not user visible but affects user visible strings. YAML: string, allowed values: driver, gpudriver, browserplugin, codec, package, software; optional, default is software.
-  url:      http://developer.download.nvidia.com/cg/Cg_3.0/license.pdf  # Url of license text to display in a web view. YAML: string.
+  url:      http://www.example.com/evil_666.0/license.pdf  # Url of license text to display in a web view. YAML: string.
   required: false       # If set to true, the user cannot proceed without accepting this license. YAML: boolean, optional, default is false.
-- id:       amd
-  name:     Catalyst
-  vendor:   "Advanced Micro Devices, Inc."
-  type:     gpudriver
-  url:      http://support.amd.com/en-us/download/eula
-  required: false
-- id:       flashplugin
-  name:     Adobe Flash
-  vendor:   Adobe Systems Incorporated
-  type:     browserplugin
-  url:      http://www.adobe.com/products/eulas/pdfs/PlatformClients_PC_WWEULA_Combined_20100108_1657.pdf
-  required: true
diff -ur calamares-2.4.1/src/modules/netinstall/netinstall.conf calamares-2.4.1-default-settings/src/modules/netinstall/netinstall.conf
--- calamares-2.4.1/src/modules/netinstall/netinstall.conf	2016-07-19 13:04:14.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/netinstall/netinstall.conf	2016-09-19 01:37:41.488266440 +0200
@@ -1,2 +1,5 @@
 ---
-groupsUrl: http://chakraos.org/netinstall.php
+# Packager note: If you want to enable the netinstall module, you will also have
+#                to create such a file (see the example in the source code).
+groupsUrl: file:///etc/calamares/modules/netinstall.yaml
+#groupsUrl: http://example.com/netinstall.yaml
diff -ur calamares-2.4.1/src/modules/packages/packages.conf calamares-2.4.1-default-settings/src/modules/packages/packages.conf
--- calamares-2.4.1/src/modules/packages/packages.conf	2016-04-12 11:23:47.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/packages/packages.conf	2016-09-19 00:35:05.068177285 +0200
@@ -11,7 +11,7 @@
 #  - portage	 - Gentoo package manager
 #  - entropy	 - Sabayon package manager
 #
-backend: packagekit
+backend: dnf
 #
 # List of maps with package operations such as install or remove.
 # Distro developers can provide a list of packages to remove
@@ -26,20 +26,7 @@
 # storage called "packageOperations" and it is processed
 # after the static list in the job configuration.
 #
-#operations:
+operations:
+  - remove:
+    - calamares
 #  - install:
-#      - pkg1
-#      - pkg2
-#  - remove:
-#      - pkg3
-#      - pkg4
-#  - install:
-#      - pkg5
-#  - remove:
-#      - pkg2
-#      - pkg1
-#    install:
-#      - pkgs6
-#      - pkg7
-#  - localInstall:
-#      - /path/to/pkg8
diff -ur calamares-2.4.1/src/modules/partition/partition.conf calamares-2.4.1-default-settings/src/modules/partition/partition.conf
--- calamares-2.4.1/src/modules/partition/partition.conf	2016-06-23 09:39:24.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/partition/partition.conf	2016-09-19 00:58:22.326638764 +0200
@@ -7,7 +7,7 @@
 ensureSuspendToDisk:    true
 
 # Correctly draw nested (e.g. logical) partitions as such.
-drawNestedPartitions:   false
+drawNestedPartitions:   true
 
 # Show/hide partition labels on manual partitioning page.
 alwaysShowPartitionLabels: true
diff -ur calamares-2.4.1/src/modules/services/services.conf calamares-2.4.1-default-settings/src/modules/services/services.conf
--- calamares-2.4.1/src/modules/services/services.conf	2016-04-12 11:23:47.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/services/services.conf	2016-09-19 00:35:05.068177285 +0200
@@ -12,6 +12,4 @@
   - name: "graphical"
     mandatory: true
 
-disable:
-  - name: "pacman-init"
-    mandatory: false
+disable: []
diff -ur calamares-2.4.1/src/modules/umount/umount.conf calamares-2.4.1-default-settings/src/modules/umount/umount.conf
--- calamares-2.4.1/src/modules/umount/umount.conf	2016-04-12 11:23:47.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/umount/umount.conf	2016-09-19 01:01:28.424629429 +0200
@@ -1,9 +1,6 @@
 ---
-#scrLog:       "/path/to/installation.log"
-#destLog:      "/var/log/installation.log"
-# example when using the Calamares created log:
-#srcLog:      "/root/.cache/Calamares/Calamares/Calamares.log"
-#destLog:     "/var/log/Calamares.log"
+srcLog:      "/root/.cache/Calamares/Calamares/Calamares.log"
+destLog:     "/var/log/calamares.log"
 # example when creating with a sudo calamares -d log:
 #srcLog:      "/home/live/installation.log"
 #destLog:     "/var/log/installation.log"
diff -ur calamares-2.4.1/src/modules/unpackfs/unpackfs.conf calamares-2.4.1-default-settings/src/modules/unpackfs/unpackfs.conf
--- calamares-2.4.1/src/modules/unpackfs/unpackfs.conf	2016-04-12 11:23:47.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/unpackfs/unpackfs.conf	2016-09-19 00:35:05.068177285 +0200
@@ -1,8 +1,5 @@
 ---
 unpack:
-    -   source: "/path/to/filesystem.img"
+    -   source: "/dev/mapper/live-base"
         sourcefs: "ext4"
         destination: ""
-    -   source: "/path/to/another/filesystem.sqfs"
-        sourcefs: "squashfs"
-        destination: ""
diff -ur calamares-2.4.1/src/modules/users/users.conf calamares-2.4.1-default-settings/src/modules/users/users.conf
--- calamares-2.4.1/src/modules/users/users.conf	2016-04-12 11:23:47.000000000 +0200
+++ calamares-2.4.1-default-settings/src/modules/users/users.conf	2016-09-19 00:35:05.068177285 +0200
@@ -3,12 +3,11 @@
     - users
     - lp
     - video
-    - network
-    - storage
+    - dialout
     - wheel
     - audio
 autologinGroup:  autologin
-doAutologin:     true
+doAutologin:     false
 sudoersGroup:    wheel
 setRootPassword: true
 doReusePassword: true
