diff -rup calamares-3.2.39-default-settings/calamares.desktop calamares-3.2.39-kdesu/calamares.desktop
--- calamares-3.2.39-default-settings/calamares.desktop	2021-03-19 09:23:08.000000000 -0400
+++ calamares-3.2.39-kdesu/calamares.desktop	2021-03-19 23:14:47.355166565 -0400
@@ -5,7 +5,7 @@ Name=Install System
 GenericName=System Installer
 Keywords=calamares;system;installer;
 TryExec=calamares
-Exec=sh -c "pkexec calamares"
+Exec=kdesu /usr/bin/calamares
 Comment=Calamares — System Installer
 Icon=calamares
 Terminal=false
diff -rup calamares-3.2.39-default-settings/calamares.desktop.in calamares-3.2.39-kdesu/calamares.desktop.in
--- calamares-3.2.39-default-settings/calamares.desktop.in	2021-03-19 09:23:08.000000000 -0400
+++ calamares-3.2.39-kdesu/calamares.desktop.in	2021-03-19 23:14:47.356166561 -0400
@@ -5,7 +5,7 @@ Name=Install System
 GenericName=System Installer
 Keywords=calamares;system;installer;
 TryExec=calamares
-Exec=sh -c "pkexec calamares"
+Exec=kdesu /usr/bin/calamares
 Comment=Calamares — System Installer
 Icon=calamares
 Terminal=false
diff -rup calamares-3.2.39-default-settings/CMakeLists.txt calamares-3.2.39-kdesu/CMakeLists.txt
--- calamares-3.2.39-default-settings/CMakeLists.txt	2021-03-19 16:01:05.087151965 -0400
+++ calamares-3.2.39-kdesu/CMakeLists.txt	2021-03-19 23:14:47.357166558 -0400
@@ -50,7 +50,6 @@ set( CALAMARES_VERSION_RC 0 )  # Set to
 ### OPTIONS
 #
 option( INSTALL_CONFIG "Install configuration files" ON )
-option( INSTALL_POLKIT "Install Polkit configuration" ON )
 option( INSTALL_COMPLETION "Install shell completions" OFF )
 # Options for the calamares executable
 option( WITH_KF5Crash "Enable crash reporting with KCrash." ON )  # TODO:3.3: WITH->BUILD (this isn't an ABI thing)
@@ -282,18 +281,6 @@ endif()
 find_package( Qt5DBus CONFIG )
 
 find_package( YAMLCPP ${YAMLCPP_VERSION} REQUIRED )
-if( INSTALL_POLKIT )
-    find_package( PolkitQt5-1 REQUIRED )
-else()
-    # Find it anyway, for dependencies-reporting
-    find_package( PolkitQt5-1 )
-endif()
-set_package_properties(
-    PolkitQt5-1 PROPERTIES
-    DESCRIPTION "Qt5 support for Polkit"
-    URL "https://cgit.kde.org/polkit-qt-1.git"
-    PURPOSE "PolkitQt5-1 helps with installing Polkit configuration"
-)
 
 # Find ECM once, and add it to the module search path; Calamares
 # modules that need ECM can do
@@ -631,13 +618,6 @@ if( INSTALL_CONFIG )
     )
 endif()
 
-if( INSTALL_POLKIT )
-    install(
-        FILES com.github.calamares.calamares.policy
-        DESTINATION "${POLKITQT-1_POLICY_FILES_INSTALL_DIR}"
-    )
-endif()
-
 if ( INSTALL_COMPLETION )
     if( NOT CMAKE_INSTALL_BASHCOMPLETIONDIR )
         set( CMAKE_INSTALL_BASHCOMPLETIONDIR "${CMAKE_INSTALL_DATADIR}/bash-completion/completions" )
