diff -ur calamares-3.2.7/CMakeModules/BoostPython3.cmake calamares-3.2.7-boost-python3/CMakeModules/BoostPython3.cmake
--- calamares-3.2.7/CMakeModules/BoostPython3.cmake	2019-04-27 20:12:03.000000000 +0200
+++ calamares-3.2.7-boost-python3/CMakeModules/BoostPython3.cmake	2019-05-05 19:42:40.290272157 +0200
@@ -15,6 +15,11 @@
 # libboost_python-3.4.so
 # depending on what python's targets you selected during install
 #
+# On Fedora >= 30 instead, the boost-python3-devel provides boost library with a
+# name like:
+# libboost_python37.so
+# depending on what python's targets you selected during install
+#
 # find_boost_python3() tries to find the package with different component
 # names. By default it tries "python3", "python-py$suffix" and
 # "python-$dotsuffix", where suffix is based on the `python_version` argument.
@@ -47,6 +52,10 @@
     _find_boost_python3_int( ${boost_version} python-py${_fbp_python_short_version} _fbp_found )
 
     if (NOT _fbp_found)
+        _find_boost_python3_int( ${boost_version} python${_fbp_python_short_version} _fbp_found )
+    endif()
+
+    if (NOT _fbp_found)
         # The following loop changes the searched name for Gentoo based distributions
         # turns "3.4.123abc" into "3.4"
         string( REGEX REPLACE "([0-9]+)\\.([0-9]+)\\..*" "\\1.\\2" _fbp_python_short_version ${python_version} )
